modules/GenericDataParser/src/client/GenericDataParser.coffee

```diff
@@ -8,3 +8,34 @@ class GenericDataParserController extends BasicFileValidateAndSaveController
 		@loadImagesFile = true
 		super()
 		@$('.bv_moduleTitle').html("Simple #{window.conf.experiment.label} Loader")
+
+		# set the number of uploads possible to one
+		@$(".fileinput-button input").attr("multiple", false)
```

Comment:
I believe this is just going to set an attribute on the input and set its value to "false".  I don't think it will remove the attribute which I think is required to make it so you can't select multiple.

Reply:
In any case, can we move this down into the LSFileChooser controller so that if someone sets max number of files to 1 then it removes the multiple attribute?